.github/workflows/build-push-images.yaml

```diff
@@ -220,4 +232,84 @@ jobs:
           password: ${{ secrets.QUAY_TOKEN }}
       - name: Print image url
         if: steps.filter.outputs.vulnerability == 'true'
-        run: echo "Image pushed to ${{ steps.push-to-quay-vulnerability-scan.outputs.registry-paths }}"
+        run: |
+          echo "Image pushed to ${{ steps.push-to-quay-vulnerability-scan.outputs.registry-paths }}"
+          echo "::set-output name=vulnerability_scan_pushed::true"
+
+      # Tag images with the latest commit ID if they are not changed.
+      - name: Tag latest commit ID to quay.io
```

Comment:
It would be cleaner to keep the steps ordered by image context, example:
- Build access-setup Image
- Push to quay.io
- Pint image url
- Tag latest commit ID to quay.io

instead of grouping all the "Tag latest commit ID to quay.io" at the end

Reply:
All the steps mentioned are ordered except for the last one as it is not certain that it will be executed each time. That's why I kept these at the end. Let me know if you still feel its better grouped at the top. I shall update it.

Reply:
I agree with Moemen, just because it will make copy pasting the actions for a new image easier (no cherry-picking from various part of the file).